src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1150,8 +1150,8 @@ public void BuildApplicationWithJavaSource (bool isRelease, bool expectedResult)
 						var runtime = runtimeInfo.FirstOrDefault (x => x.Abi == abi && x.Runtime == expectedRuntime);
 						Assert.IsNotNull (runtime, "Could not find the expected runtime.");
 						var inApk = ZipHelper.ReadFileFromZip (apk, String.Format ("lib/{0}/{1}", abi, runtime.Name));
-						var inApkRuntime = runtimeInfo.FirstOrDefault (x => x.Abi == abi && x.Size == inApk.Length);
-						Assert.IsNotNull (inApkRuntime, "Could not find the actual runtime used.");
+						var inApkRuntime = runtimeInfo.FirstOrDefault (x => x.Abi == abi && x.Runtime == expectedRuntime);
```

Comment:
we use x.Size because we want to know what runtime made it into the apk. We can't do that via "x.Runtime" because we don't have that into from the `inApk` variable. We only have the size of the .so in the apk. 
The idea is to make sure the thing in the apk is the one we expected.

Reply:
perhaps its better to just asset on `runtime.Size == inApk.Length`

Reply:
Ok, thanks now I understand a bit more on what the test is trying to do.

Something is weird here, I just debugged one as an example `CheckSequencePointGeneration(False,False,False,True,"Full",True,"debug")`.

`inApk.Length` is 157600, and when I look at the matching abis in `runtimeInfo`:

![image](https://user-images.githubusercontent.com/840039/32257818-47b200f6-be85-11e7-9ef6-2a8a867211d3.png)

It seems like on Windows it's somehow like 25K bigger? Is that weird?

Maybe we should just wait on some of these until next week. It might be easier to fix them in person.